core/templates/components/skill-selector/select-skill-modal.component.spec.ts

```diff
@@ -94,7 +93,8 @@ describe('Select Skill Modal', () => {
   it('should close modal on confirm', () => {
     spyOn(ngbActiveModal, 'close');
     componentInstance.selectedSkillId = '2';
-    let totalSkills = [];
+    let totalSkills:
+    (ShortSkillSummary |SkillSummaryBackendDict)[] = [];
```

Comment:
Actually it is because we have two types of skill summaries to deal with in this component: 
the `SkillSummaryBackendDict` which is an interface for `SkillSummary` class.

The `ShortSkillSummary` that has only two properties of the `SkillSummary` class
Therefore there was need to assign the type of `totalSkills` to an array of both types

Reply:
Hmm, the formatting is still a bit wrong, but I don't understand why one of these is a dictionary and the second is not? Is it just wrong naming?